pom.xml

```diff
@@ -4,8 +4,8 @@
 
     <parent>
         <groupId>guru.nidi</groupId>
-        <artifactId>guru-nidi-parent-pom</artifactId>
-        <version>1.0.5</version>
+        <artifactId>guru-nidi-parent-pom-jdk6</artifactId>
```

Comment:
This breaks the build because this artifact doesn't exist. What did you change in it? Only the `java.version` property? Maybe you can override it locally.

This said, if Stefan wants to support JDK 6, I think using a qualifier to release a `jdk6` flavour of `guru-nidi-parent-pom` would be more appropriate.